rules/docker-gnmi.mk

```diff
@@ -9,6 +9,7 @@ $(DOCKER_GNMI)_PATH = $(DOCKERS_PATH)/$(DOCKER_GNMI_STEM)
 $(DOCKER_GNMI)_DEPENDS += $(SONIC_MGMT_COMMON)
 $(DOCKER_GNMI)_DEPENDS += $(SONIC_TELEMETRY)
 $(DOCKER_GNMI)_DBG_DEPENDS = $($(DOCKER_CONFIG_ENGINE_BOOKWORM)_DBG_DEPENDS)
+$(DOCKER_GNMI)_PYTHON_WHEELS = $(SONIC_PLATFORM_COMMON_PY3) $(SONIC_UTILITIES_PY3)
```

Comment:
@wen587 , FYI:

I don't think that it is a good idea to import/duplicate the sonic-utlitities packages as a whole within gnmi. If our goal is to move GCU to GNMI then we should just import GCU specific files to docker-gnmi, rather than all of sonic-utilities package.

What do you think? Gang suggested that it may be better, but the challenge is that we need to build GCU package separately. I think that's a good investment.